rankers/SimpleAggregateRanker/manifest.yml

```diff
@@ -0,0 +1,13 @@
+manifest_version: 1
+name: SimpleAggregateRanker
+kind: ranker
+type: pod
+description: :class:`SimpleAggregateRanker` calculates the score of the matched doc from the matched chunks.
+  For each matched doc, the score is aggregated from all the matched chunks belonging to that doc.
+author: Jina AI Dev-Team (dev-team@jina.ai)
+url: https://jina.ai
+vendor: Jina AI Limited
+documentation: https://github.com/jina-ai/jina-hub
+version: 0.0.6
+license: apache-2.0
+keywords: [ranker, toyexample, demo]
```

Comment:
toyexample and demo seem like bad `keywords`

Reply:
True, I thought it has to be like this because it was the same in Min/MaxRanker.
I updated the keywords. Thanks.